### PR TITLE
Bullseye Release findings: NetworkManager + KMS Driver

### PR DESCRIPTION
**Pull request by nbuchwitz**
Use separate config files for link local fallback for eth0 and eth1. Also explicitly set dhcp timeout and retries for DHCP interface (eth0).

This fixes connection problems on RevPi devices with more than one connected interface, where only the one interface came up correctly configured. The profile was only used for one interface and the other left unconfigured.

---

Also disable randomized WLAN mac addresses

### CODE REVIEW

**Review by iluminat23 — APPROVED** [review]

**Review by naruxde — CHANGES_REQUESTED** [review]
Shouldn't we still prepare / carry out the installation of the backport?

**Comment by nbuchwitz** [comment]
After lots of testing with backported NetworkManager there was always something which was not working as desired. The final solution is to use my original approach which states the desired functionality in three explicit config files.

The desired approach is as following:
1. Try to acquire an IP address via DHCP
2. If DHCP was successful use this IP and try to configure an IPv6 with DHCPv6 / SLAAC / LL
2. If DHCP fails fallback to link local (both, IPv4 and IPv6

We should document this configuration in our release notes.

The current patchset was successfully tested on a Connect S and Connect 4 (with working DHCP and LL only, both interfaces).

File: templates/network-manager/fallback-dhcp.nmconnection [hunk 1]
```diff
@@ -2,11 +2,16 @@
...
+interface-name=eth0
+autoconnect=yes
 autoconnect-priority=-98
-permissions=
+autoconnect-retries=2
 
 [ipv4]
 method=auto
+may-fail=no
+dhcp-timeout=2
```
Comment (naruxde) [inline templates/network-manager/fallback-dhcp.nmconnection:13]:
```suggestion
dhcp-timeout=2
```
Could be to short, if users have a DHCP-relay Server, that will fail... Let's think about 15-20?

File: templates/network-manager/fallback-dhcp.nmconnection [hunk 2]
```diff
@@ -2,11 +2,16 @@
 id=DHCP on unbound interfaces
```
Comment (naruxde) [inline templates/network-manager/fallback-dhcp.nmconnection:2]:
```suggestion
id=DHCP on eth0 interface
```

**Review by naruxde — CHANGES_REQUESTED** [review]

**Comment by naruxde** [comment]
Let's rename `fallback-dhcp.nmconnection`to something like `dhcp-eth0.nmconnection`.

File: templates/network-manager/fallback-dhcp.nmconnection (hunk 1 above)
Comment (nbuchwitz) [reply templates/network-manager/fallback-dhcp.nmconnection]: Would be fine with me. Nevertheless this must be documented in our release notes / changes, as this is the delay after the LL fallback will settle.

Comment (nbuchwitz) [reply templates/network-manager/fallback-dhcp.nmconnection]: PR is updated

**Review by naruxde — APPROVED** [review]

**Pull request merged**